tests/quarkus/src/main/resources/application.properties

```diff
@@ -27,7 +27,7 @@ quarkus.security.users.embedded.users.hawtio = hawtio
 quarkus.security.users.embedded.roles.hawtio = admin
 
 # 'sample' custom plugin settings
-quarkus.hawtio.plugin.sample-plugin.url = http://localhost:8081
+#quarkus.hawtio.plugin.sample-plugin.url = http://localhost:8080
```

Comment:
Maybe it's better to add a comment above the commented line about when this parameter should be used.